checkers/Board.py

```diff
@@ -267,14 +491,15 @@ def render(self, screen, player_turn):
 
                     if self.grid[row][col].player == 1:
                         color = self.colors['WHITE']
-                    elif self.grid[row][col].player == -1:
+                    
+                    if self.grid[row][col].player == -1:
                         color = self.colors['BLACK']
 
                     pygame.draw.circle(
                         screen, color,
                         (col * self.grid_size + self.grid_size / 2,
                          row * self.grid_size + self.grid_size / 2),
-                        self.grid_size / 3)
+                        self.grid_size / 3, draw_top_right = self.grid[row][col].status == 'king')
```

Comment:
I assume this is not the final king rendering right? Any ideas for king rendering?

Reply:
Yes. It's a tempary implementation.